### ISSUE TITLE
Microsoft Outlook and Teams "browser version isn't supported"

### ISSUE THREAD

**Issue by zworkinspace** [ferdium/ferdium-app#1763]
### Avoid duplicates

- [X] I agree to follow the [Code of Conduct](https://github.com/ferdium/ferdium-app/blob/HEAD/CODE_OF_CONDUCT.md) that this project adheres to.
- [X] I have searched the [issues tracker](https://github.com/ferdium/ferdium-app/issues?q=is%3Aopen+is%3Aissue+label%3A%22bug+%3Abeetle%3A%22) for a bug report similar to mine, in vain

### Ferdium Version

6.7.3

### What Operating System are you using?

macOS

### Operating System Version

13.6.6

### What arch are you using?

x64

### Expected Behavior

When adding a Teams (or Outlook) service and entering valid credentials the Teams page should load.

### Actual Behavior

The Teams credentials (including 2FA) are successfully entered and appear to be accepted but an error message page is returned: 
"Hmm. Your browser version isn't supported. ... "

### Steps to reproduce

1. Click on add service.
2. Select Microsoft Teams.
3. Enter User Agent: https://teams.microsoft.com
4. Leave other settings to defaults.
5. Save Service
6. Enter credentials on Sign in panel for a valid Teams / O365 account

### Additional information

This may be entirely due to 0365 refusing to serve to other than supported Chromium browsers (Edge or Google Chrome), Safari or Firefox, and not something that can be fixed on the ferdium client side - ??  

Might there be a way to have our 0365 cloud service provider configure the server to accept connections from ferdium's client?

Otherwise, THANKS for a great app that at least lets one integrate Slack, Mattermost and many other services to help reduce the desktop Tower of Babel problem.

**Comment by zworkinspace** [comment]
Note:  rambox has exactly the same problem.

**Comment by SpecialAro** [comment]
Hello @zworkinspace, thank you for logging this issue.

Just to be clear, are you adding an URL to the user agent?

> Enter User Agent: https://teams.microsoft.com

This is NOT how you change the user agent (and you shouldn't be doing it in the first place if the service works without it) - if that is the case, no wonder the service complains it doesn't know the browser you are in (you are saying you are using https://team.microsoft.com browser) 🙈

Let me know if you need further help to fix this!
If it is still an issue I think I might be able to find a workaround.

**Comment by zworkinspace** [comment]
well shoot, as I was going through the service setup it looked like I had to enter the URL similar to other services, it didn't occur to me to just create the service and open it!!  Perhaps the documentation should make that clearer?

Upon deleting the Teams service that didn't work and creating a new one, leaving all parameters at default and simply opening it, I got to the login and was able to enter my account 2FA credentials. 

Now I've got my assorted Slacks, Mattermosts and Teamses all in one place, slightly improving the chances I won't miss smoke signals on them.

Thanks, @SpecialAro , for your very prompt response and solution!
